deploy/platform/kubernetes/Makefile

```diff
@@ -155,9 +156,10 @@ feature-mesh-with-agent:
 deploy: $(features)
 	helm dep up .
 	helm -n $(NAMESPACE) upgrade --install $(RELEASE) . --create-namespace --timeout=20m $(HELM_OPTIONS)
-
+ifneq ($(DEBUG),true)
```

Comment:
Does only Istio-related sidecar automatically injected?

Reply:
What do you mean? This is nothing related to Istio

Reply:
I'm curious about the purpose of this step (rollout restart).

Reply:
make sense